examples-chart/templates/deployments/pre-install-rbac.yaml

```diff
@@ -30,9 +30,6 @@ rules:
   - apiGroups: ["applicationconnector.kyma-project.io"]
     resources: ["eventactivations"]
```

Comment:
None of the pods are using `eventactivations`.  Does it make sense remove this as well?

Reply:
You're right. Removed.